src/middleware/isnt-webhook.ts

```diff
@@ -1,10 +1,10 @@
-module.exports = isntWebhook;
-
 // Example webhook event request:
 // https://developer.github.com/webhooks/#example-delivery
-function isntWebhook(request, options) {
+import { Request } from "express";
+
+export function isntWebhook(request: Request, options: { path?: string }) {
   // GitHub sends all events as POST requests
-  if (request.method !== "POST") {
+  if (request.method !== "POST" || !request.url) {
```

Comment:
why the additional check?

Reply:
> why the additional check?

Mmm... at some point I was getting coverage issues. I tried to reproduce it now to share it but... not happening anymore so... removing this extra unnecessary check :)